forte/processors/data_augment/algorithms/UDA.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This file implements the UDA(Unsupervised Data Augmentation)
+algorithm as described in
+Unsupervised Data Augmentation for Consistency Training
+(https://arxiv.org/abs/1904.12848)
+(https://github.com/google-research/uda)
+"""
+import tensorflow as tf
+
+def kl_divergence_log_probs(log_p, log_q):
```

Comment:
Just to confirm: Is it `log probability` or `logits`? 

We may name it `kl_divg_with_log_probs`, or `kld_with_log_probs`, for conciseness and conformity to the convention, e.g., `binary_cross_entropy_with_logits` ([here](https://github.com/asyml/texar-pytorch/blob/master/texar/torch/losses/mle_losses.py#L273))